saleor/account/emails.py

```diff
@@ -24,12 +30,14 @@ def send_password_reset_email(context, recipient):
 @shared_task
 def send_account_delete_confirmation_email(token, recipient_email):
     site = Site.objects.get_current()
-    absolute_url = build_absolute_uri(location=None)
+    logo_url = build_absolute_uri(
```

Comment:
Could you refactor a little the code remove the repeated code? For example, have ` get_email_base_context()` to return the context (`{'sitename': ..., 'domain': ..., 'logo_url':: ...}`) then you add whatever you need to the retrieved base context.

That would be great :+1: